rustls/src/verify.rs

```diff
@@ -539,14 +538,20 @@ fn prepare<'a, 'b>(
 /// certificate, without any name checking.
 pub struct AllowAnyAuthenticatedClient {
     roots: RootCertStore,
+    subjects: Vec<Vec<u8>>,
 }
 
 impl AllowAnyAuthenticatedClient {
     /// Construct a new `AllowAnyAuthenticatedClient`.
     ///
     /// `roots` is the list of trust anchors to use for certificate validation.
     pub fn new(roots: RootCertStore) -> Self {
-        Self { roots }
+        let subjects = roots
```

Comment:
Style nit: I'd avoid binding this separately, inlining this into the initialization below.